update_to_latest.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Good idea this script, but I would add a small comment at the top of what it does and how to use it. Also maybe add a small topic in the readme file about this.